example-data/turtle/sio-model/consent.ttl

```diff
@@ -2,17 +2,24 @@
 @prefix sio: <http://semanticscience.org/resource/> .
 @prefix obo: <http://purl.obolibrary.org/obo/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-@prefix prov: <http://www.w3.org/ns/prov#> .
-
 
 :person_ a sio:SIO_000498 ;
-    sio:has-role :role_ .
+    sio:SIO_000228 :role_1, :role_2 .
+
+:role_1 a obo:OBI_0000093 ;
+    sio:SIO_000356 :process_1 .
 
-:role_ a obo:OBI_0000093 ;
-    sio:SIO_000356 :process_ .
+:process_1 a obo:ICO_0000149 ;
+    sio:SIO_000229 :output_1 ;
+    sio:SIO_000680 "2020-10-18T13:00:00"^^xsd:dateTime .    
+    
+:role_2 a obo:OBI_0000093 ;
+    sio:SIO_000356 :process_2 .
 
-:process_ a obo:ICO_0000196 ;
-    sio:SIO_000229 <http://example.org/my_consent_document.pdf> ;
-    prov:startedAtTime "2020-10-18T13:00:00"^^xsd:dateTime .
+:process_2 a obo:ICO_0000181 ;
+    sio:SIO_000229 :output_2 ;
+    sio:SIO_000680 "2020-10-18T13:03:00"^^xsd:dateTime .
+    
+:output_1 a sio:SIO_000785, obo:NCIT_C49488 .
 
-<http://example.org/my_consent_document.pdf> a sio:SIO_000340, obo:OBIB_0000488 .
+:output_2 a sio:SIO_000785, obo:NCIT_C49487.
```

Comment:
Same as before. For the next release, as alternative to using NCIT could be: 1) 'no answer to question' (obo:OPMI_0000024), which is an information entity in the Ontology of Precision Medicine and Investigation; and optionally refer to 'full disagreement' (sio:001181), which is an informational quality:

'no answer to question' (http://purl.obolibrary.org/obo/OPMI_0000024)
 subClassOf
'information content entity'
that 'has value' value "no"
and 'refers to' some 'agreement quality'

'full disagreement'
subClassOf
'agreement quality'

'person'
subClassOf
'entity'
and 'has quality' some 'agreement quality'


In RDF:
:person_ a sio:person;
  sio:has-quality sio:full-disagreement .

:output_ a opmi:no-answer-to-question;
   sio:refers-to sio:full-disagreement .